app/scripts/TiledPlot.js

```diff
@@ -1987,7 +1987,8 @@ class TiledPlot extends React.Component {
           initialXDomain={this.props.initialXDomain}
           initialYDomain={this.props.initialYDomain}
           isRangeSelection={this.props.mouseTool === MOUSE_TOOL_SELECT}
-          leftWidth={this.leftWidth}
+          isRisShowGlobalMousePosition={this.props.isShowGlobalMousePosition}
```

Comment:
what is the `Ris` in `isRisShowGlobalMousePosition`?

Reply:
Ohh shoooot. Good catch! This is derived from a merge conflict I believe